src/solver/simulation/timeseries-numbers.cpp

```diff
@@ -810,68 +660,6 @@ void applyMatrixDrawsToInterModalModesInArea(Matrix<uint32_t>* tsNumbersMtx,
     }
 }
 
-// Set tsNumbers to 1 for all years if only one TS is present
-static void fixTSNumbersSingleAreaSingleMode(Matrix<uint32_t>& tsNumbers, uint width, uint years)
-{
-    if (width == 1)
-    {
-        for (uint year = 0; year < years; year++)
-        {
-            tsNumbers[0][year] = 0;
-        }
-    }
-}
-
-static void fixTSNumbersWhenWidthIsOne(Study& study)
```

Comment:
**Explanations** : now we that we made **zero** a  default value (at construction) for TS numbers, this makes these previous fix functions useless.